### DISCUSSION TITLE
I am writing client for android

### DISCUSSION THREAD

**Discussion by konachan700** [photoprism/photoprism#2515] [Show and tell]
Hello!
I started to write android client for Photoprism:
https://github.com/konachan700/MeW-PhotoPrism

**Comment by centralhardware** [comment]
looks grate, waiting for beta

**Comment by graciousgrey** [comment]
Cool :)

Once there is a beta version for testing, we can add a link to your repo in 

https://github.com/photoprism/photoprism-contrib 

if you like.

Please make sure you follow our brand guidelines. It would be great if you would use something like "Android client for PhotoPrism" or "Android client compatible with PhotoPrism" in your readme file and about text instead of "PhotoPrism client for Android". 

https://photoprism.app/trademark

Thank you very much!

**Comment by konachan700** [comment]
> It would be great if you would use something like "Android client for PhotoPrism" or "Android client compatible with PhotoPrism"

fixed

**Comment by MapZombie** [comment]
> Hello! I started to write android client for Photoprism: https://github.com/konachan700/MeW-PhotoPrism

Is this now dead? Any updates?

**Comment by konachan700** [comment]
no, but I have no enough time for it now
